website_sale_embargo/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# © initOS GmbH 2023
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Website Sales Embargo",
+    "version": "15.0.1.0.0",
+    "depends": [
+        "product_harmonized_system",
+        "sale",
+        "website_sale",
+    ],
+    "author": "initOS GmbH",
+    "category": "",
+    "summary": """
+        Add Embargo to Product's HS Code
+    """,
+    "website": "https://github.com/OCA/server-tools",
```

Comment:
That's certainly not the right repository.

Reply:
@jans23 oh yes it is not, sorry. OCA/e-commerce?

Reply:
Either "e-commerce" or "sale-workflow"?